docs/gui/cult-lists.rst

```diff
@@ -0,0 +1,58 @@
+gui/cult-lists.lua
```

Comment:
There were several comments left on the original PR (https://github.com/DFHack/scripts/pull/664/files), and it looks like many could be addressed here as well. This line in particular needs the `.lua` suffix removed, for one thing.